runtime/memory/64bit_nix.c

```diff
@@ -41,22 +41,10 @@ void expand_memory() {
     memory_size += WASM_PAGE_SIZE;
 }
 
-i32 instruction_memory_size() {
-    return memory_size / WASM_PAGE_SIZE;
-}
-
-i32 instruction_memory_grow(i32 count) {
-    i32 prev_size = instruction_memory_size();
-    for (int i = 0; i < count; i++) {
-        expand_memory();
-    }
-
-    return prev_size;
-}
-
 INLINE void check_bounds(u32 offset, u32 bounds_check) {
     // Due to how we setup memory for x86, the virtual memory mechanism will catch the error, if bounds < WASM_PAGE_SIZE
-    awsm_assert(bounds_check < WASM_PAGE_SIZE || (memory_size > bounds_check && offset <= memory_size - bounds_check));
+    awsm_assert(bounds_check < WASM_PAGE_SIZE
+                || (memory_size > bounds_check && offset <= memory_size - bounds_check)); ////// why not offset ?
```

Comment:
This question is without context. Please add it, and elucidate any concerns you have.

Reply:
This was a question to myself. Forgot to remove it :/